README-CVM.md

```diff
@@ -0,0 +1,39 @@
+## How to build a CVM compatiable image
+
+```sh
+git clone https://github.com/linuxkit/linuxkit.git
+cd linuxkit
+git fetch origin pull/3947/head:cvm && git checkout -b cvm
```

Comment:
This mostly should go away. This describes how to do it as long as the PR is open. The assumption of the PR is that this will merge into master soon.